indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -252,21 +254,22 @@ TimetableSessionsInput.defaultProps = {
 export function TimetableSessionsSettings() {
   return (
     <>
-      <Form.Group inline>
-        <label>{Translate.string('Default display for sessions')}</label>
-        <FinalRadio name="display" label={Translate.string('Expanded dropdown')} value="expanded" />
-        <FinalRadio
-          name="display"
-          label={Translate.string('Collapsed dropdown')}
-          value="collapsed"
-        />
-      </Form.Group>
+      <FinalDropdown
+        name="display"
+        label={Translate.string('Default display for sessions')}
```

Comment:
After looking at the behavior of this setting again: Do we really need this to be a string option? I can't imagine any other future display options that would fit there. So maybe this should be a "Expand days by default" checkbox instead of a radio or dropdown?

Reply:
While I like the idea of a simple checkbox, this triggers a number of usability questions. Should this checkbox be checked by default on new fields? I believe that this will be the most common setting. If so, the wording makes me think that expanding days by default is the special case. Perhaps inverting the wording and logic would make more sense (i.e. "Collapse days by default" and leaving the checkbox unchecked by default in new fields).

But the more I think about this, the more I question this feature altogether. Having the list of session blocks collapsed by default will leave the actionable UI elements hidden and will make it easy for registrants to miss the field. Would it make sense to remove this setting?

@jbtwist, it may be worth to do a quick check in our DB and see if many of our registration forms are configured so that session blocks are collapsed by default.

Reply:
Regardless of whether it's a checkbox or something else I think we should indeed agree on a decent default for that setting.

Reply:
> I would keep the string since boolean or numeric values gives no information about which option have been selected.

I would not keep it named `display` but rather rename it to `collapse_days` - and then a boolean makes much more sense.

The main reason why I'd prefer a boolean is that I can't think of a case where we'd add a third display option that would make the existing two obsolete - so when adding another display-related setting, it'd probably make more sense for it to be a separate option instead of another value for the old "display".